libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewApi.kt

```diff
@@ -105,4 +107,11 @@ class NavigationViewApi internal constructor(
     fun isReplayEnabled(): Boolean {
         return store.state.value.tripSession.isReplayEnabled
     }
+
+    /**
+     * Get access to [MapView] instance created by [NavigationView].
+     */
+    fun getMapView(): MapView? {
```

Comment:
Will you make this observable? It seems we should be able to attach and detach.

Otherwise it's easy to create dangling subscriptions after you update the mapview

Reply:
Fixed.